examples/adts.scala

```diff
@@ -0,0 +1,17 @@
+package examples
+
+object adts:
+  enum Rgb:
+    case Red
+    case Green
+    case Blue
+
+  enum Bool:
+    case True
+    case False
+
+  final case class RgbPair(snd: Bool, fst: Rgb)
+
+  def idRgb(theArg: Rgb): Rgb = theArg
+
+  def constRgbPair(rgbArg: Rgb, rgbPairArg: RgbPair): RgbPair = rgbPairArg
```

Comment:
I keep output of of compiling [examples/adts.agda](https://github.com/lemastero/agda2scala/blob/main/examples/adts.agda) to Scala 3 in [scala3/src/main/scala/examples/adts.scala](https://github.com/lemastero/agda2scala/blob/main/scala3/src/main/scala/examples/adts.scala).

That is what command described in README:
```shell
cabal run -- agda2scala --compile --no-main --scala-dialect=Scala3 --out-dir=scala3/src/main/scala ./examples/adts.agda
```
does. Can you generate and commit it instead of this one?